content/articles/building-model-using-code-first/index.md

```diff
@@ -1,4 +1,4 @@
-There are three workflows to build a domain model using Entity Framework. These are, `database first`, `Code first`, and `model first`.
+There are three workflows used to build a domain model using Entity Framework. These are, `database first`, `Code first`, and `model first`.
 
 With the `database first`, we start with the database. We design our tables using visual designers. The EntityFramework generates the domain classes based on the database. 
```

Comment:
These frameworks shouldn't all begin with the word "With". Find a way to a make the sentences continuous.